build.rs

```diff
@@ -410,34 +414,60 @@ fn build_unix(target_info: &TargetInfo, out_dir: PathBuf) {
         .any(|p| need_run(&p, test_target)) ||
                              lib_header_change;
 
-    let mut additional = Vec::new();
-    for additional_src in target_info.srcs() {
-        for src in additional_src.iter() {
-            additional.push(make_asm(src, out_dir.clone(), &target_info));
+    // copied from gcc
+    let mut cfg = rayon::Configuration::new();
+    if let Ok(amt) = env::var("NUM_JOBS") {
+        if let Ok(amt) = amt.parse() {
+            cfg = cfg.set_num_threads(amt);
         }
     }
+    drop(rayon::initialize(cfg));
```

Comment:
This code is coming from the [gcc crate](https://github.com/alexcrichton/gcc-rs/blob/master/src/lib.rs#L372-L378). I'm not sure why drop is called here. (I think it should not be needed)